Backend/Models/User.model.js

```diff
@@ -1,11 +0,0 @@
-const mongoose = require('mongoose');
-
-const UserSchema = mongoose.Schema({
-    email :{type : String, required : true},
-    password : {type : String, required : true},
```

Comment:
how `userId` is referred to in  TimerSchema  and TaskSchema 
as there is no userId preset in UserSchema?

userId is a JWT token?